meta-mbl-bsp-common/recipes-bsp/atf/files/0001-imx-imx6-uboot_ddr_init-Import-u-boot-DDR-init-code.patch

```diff
@@ -0,0 +1,1650 @@
+From 97faca4a323b99aa4d0c3c741ab0856f7a2e15a0 Mon Sep 17 00:00:00 2001
```

Comment:
This file should be moved to `meta-freescale-3rdparty-mbl/recipes-bsp/atf/files`